cli/tests/test_workspaces.rs

```diff
@@ -887,15 +884,15 @@ fn test_workspaces_current_op_discarded_by_other(automatic: bool) {
         let output = test_env.run_jj_in(&secondary_path, ["st"]);
         insta::assert_snapshot!(output, @r"
         Working copy changes:
-        A added
+        C {modified => added}
```

Comment:
Is this expected?

Reply:
Same as above

Reply:
Is it normal that `A` was mentioned before and now is not?

Reply:
Yes, `C` implies that a file was created/added. We never (yet, at least) indicate a copy to a file that existed before, so no "copy into existing file".